### PR TITLE
Add wayfinding to Landscape, clean up Help & Support

### PR DESCRIPTION
**Pull request by patosullivan**
Fixes https://github.com/tloncorp/landscape-apps/issues/1856 for landscape itself.

Includes new calm engine pref for showing/hiding the wayfinding toggle.

Also cleaned up the Help & Support section of system prefs, removed the link to rondev's group discovery group and replaced it with a link to Tlon Public.

localhost_3001_apps_grid_leap_system-preferences_privacy
localhost_3001_apps_grid_leap_system-preferences_privacy (1)
localhost_3001_apps_grid_
localhost_3001_apps_grid_ (1)
localhost_3001_apps_grid_ (2)
localhost_3001_apps_grid_(iPhone XR)

### CODE REVIEW

**Review by urcades — APPROVED** [review]
🫡

File: ui/src/components/LandscapeWayfinding.tsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    link: '/apps/groups/groups/~natnex-ronret/door-link',
+  },
+  tlonPublic: {
+    title: 'Tlon Public',
+    description: 'A place to ask for help',
+    icon: 'https://sfo3.digitaloceanspaces.com/zurbit-images/dovsem-bornyl/2022.6.16..19.11.20-flooring.jpeg',
+    color: 'bg-yellow-500',
+    link: '/apps/groups/groups/~nibset-napwyn/tlon',
+  },
+};
```
Comment (arthyn) [inline ui/src/components/LandscapeWayfinding.tsx:47]: would be nice to maybe just use the light groups scry for this so we don't have to hardcode it

File: ui/src/nav/Help.tsx [hunk 2]
```diff
@@ -14,53 +14,40 @@ const groups: Record<string, Group> = {
...
   tlonSupport: {
     title: 'Tlon Support Forum',
-    icon: '',
+    icon: 'https://sfo3.digitaloceanspaces.com/zurbit-images/dovsem-bornyl/2022.12.14..20.02.15-tlonstarpng.png',
     color: 'bg-yellow-500',
-    link: '/apps/groups/groups/~bitpyx-dildus/tlon-support'
-  }
+    link: '/apps/groups/groups/~bitpyx-dildus/tlon-support',
+  },
+  tlonPublic: {
```
Comment (arthyn) [inline ui/src/nav/Help.tsx:43]: same comment here, would be nice to just pull live-ish data

File: ui/src/components/LandscapeWayfinding.tsx [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+  return (
+    <Popover.Root>
+      <div className="absolute left-4 bottom-16 z-50">
+        <Popover.Trigger className="relative" asChild>
+          <button className="h-9 w-9 cursor-pointer rounded-lg bg-black text-xl text-white">
+            ?
+          </button>
+        </Popover.Trigger>
+        <Popover.Content
```
Comment (arthyn) [inline ui/src/components/LandscapeWayfinding.tsx:165]: same comment as other pr, why not use dropdown?

**Review by arthyn — APPROVED** [review]
lgtm! nothing blocking just a question and maybe some nice to haves

**Pull request merged**